src/main/java/mate/academy/Main.java

```diff
@@ -6,16 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            logger.error("Failed login attempt. Reason: {}", e.getMessage(), e);
```

Comment:
The log message here is informative, but make sure all log messages throughout the project are similarly detailed, especially in the `AuthenticationServiceImpl` and `OrderServiceImpl` classes .